licenses/LICENSE.source.json.json

```diff
@@ -7,13 +7,13 @@ All rights reserved.
 MIT License
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
-documentation files (the ""Software""), to deal in the Software without restriction, including without limitation 
-the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, 
+documentation files (the ""Software""), to deal in the Software without restriction, including without limitation
```

Comment:
Is it safe to edit this file? I thought it has something to do with JSON format...

Reply:
Now that I see the double quote on that file I realise that this whitespace may be needed. Time to use git blame to find who added these and what is their purpose...

Reply:
never intended to be valid json files, the extention got carried over from the source file name. 

these were renamed in #682